lib/src/signature.dart

```diff
@@ -145,7 +146,38 @@ class EOSSignature extends EOSKey {
     BigInt eNeg = (-e) % n;
     BigInt rInv = r.modInverse(n);
 
-    ECPoint Q = (R * s + G * eNeg) * rInv;
+    ECPoint Q = multiplyTwo(R, s, G, eNeg) * rInv;
     return Q;
   }
+
+  static bool testBit(BigInt j, int n) {
+    return (j >> n).toUnsigned(1).toInt() == 1;
+  }
+
+  static ECPoint multiplyTwo(ECPoint t, BigInt j, ECPoint x, BigInt k) {
+    var i = max(j.bitLength, k.bitLength) - 1;
```

Comment:
Could you change the var to the actual type?

Reply:
fixed.

Reply:
thx. Could you move this sample to the test?

Reply:
added `test/sign_string_test.dart` with 4 cases.

This PR fix #2. 

#3 and #4 is still failed.